modules/imaging_browser/ajax/setSortedRows.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Used to keep the sorted order of rows in imaging_browser menu for
+ * the navigation links in the imaging viewer
+ *
+ * PHP Version 5
+ *
+ * @category Documentation
+ * @package  Main
+ * @author   David Blader <dblader.mcin@gmail.com>
+ * @license  Loris license
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+// Get LORIS user issuing the request
+$user =& User::singleton();
+if (!($user->hasPermission('imaging_browser_view_allsites')
+    || ($oneIsStudySite
+    && $user->hasPermission('imaging_browser_view_site'))
+    || $user->hasPermission('imaging_browser_phantom_allsites')
+    || $user->hasPermission('imaging_browser_phantom_ownsite'))
+) {
+    header("HTTP/1.1 403 Forbidden");
+    exit;
+}
+
+set_include_path(
+    get_include_path().":../../project/libraries:../../php/libraries:"
+);
```

Comment:
I feel like this is already done in the AjaxHelper base class. @driusan can you confirm?